src/service/viewport/viewport-binding-ios-embed-wrapper.js

```diff
@@ -234,14 +234,12 @@ export class ViewportBindingIosEmbedWrapper_ {
 
   /** @override */
   getContentHeight() {
-    // The reparented body inside wrapper will have the correct content height.
-    // Body is overflow: hidden so that the scrollHeight include the margins of
-    // body's first and last child.
-    // Body height doesn't include paddingTop on the parent, so we add on the
-    // position of the body from the top of the viewport and subtract the
-    // scrollTop (as position relative to the viewport changes as you scroll).
-    const rect = this.win.document.body./*OK*/getBoundingClientRect();
-    return rect.height + rect.top + this.getScrollTop();
+    const scrollingElement = this.wrapper_;
+    const style = getComputedStyle(scrollingElement);
+    return scrollingElement./*OK*/scrollHeight
```

Comment:
Shouldn't we use `getBoundingClientRect().height` instead of `scrollHeight`? This is to avoid the issue Peter tried to fix in #12229.

Reply:
Done.